src/bindings/python/src/pyopenvino/frontend/extension.cpp

```diff
@@ -56,6 +57,45 @@ void regclass_frontend_JsonConfigExtension(py::module m) {
     }));
 }
 
+void regclass_frontend_ConversionExtensionBase(py::module m) {
+    py::class_<ConversionExtensionBase, ConversionExtensionBase::Ptr, ov::Extension> ext(m,
+                                                                                         "ConversionExtensionBase",
+                                                                                         py::dynamic_attr());
+}
+
+void regclass_frontend_ConversionExtension(py::module m) {
+    py::class_<ConversionExtension, ConversionExtension::Ptr, ConversionExtensionBase> _ext(m,
+                                                                                            "_ConversionExtension",
+                                                                                            py::dynamic_attr(),
+                                                                                            py::module_local());
```

Comment:
looks like the classes should be locally registered here because they have different names (e.g. _ConversionExtension, _ConversionExtensionONNX)
ConversionExtension is a different class in each extension.cpp file, so I have add the suffixes ONNX, TF,  Paddle
but I forgot to remove py::module_local

Reply:
py::module_local() calls have been deleted